client/my-sites/earn/home.tsx

```diff
@@ -100,14 +102,19 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 		} );
 	};
 
-	const getPlanNames = () => {
-		const jetpackText = shouldShowOfferResetFlow()
-			? ''
-			: ' ' + translate( 'Available only with a Premium or Professional plan.' );
+	const getAnyPlanNames = () => {
+		const jetpackText = translate( 'Available with Jetpack Security and Jetpack Complete.' );
 
 		// Space isn't included in the translatable string to prevent it being easily missed.
 		return isNonAtomicJetpack
-			? jetpackText
+			? ' ' + jetpackText
+			: ' ' + translate( 'Available with any paid plan.' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Available with any paid plan' )` *ES Score: 10*